test/data/testinit.js

```diff
@@ -48,6 +49,32 @@ QUnit.assert.t = function( a, b, c ) {
 	}
 
 	this.deepEqual( f, q.apply( q, c ), a + " (" + b + ")" );
+}
+
+/**
+ * Asserts that a select matches the given IDs.
+ * The select is not bound by a context.
+ * @param {String} a - Assertion name
+ * @param {String} b - Sizzle selector
+ * @param {String} c - Array of ids to construct what is expected
+ * @example t("Check for something", "//[a]", ["foo", "bar"]);
+ * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ */
+QUnit.assert.t = function( a, b, c ) {
+	match( a, b, c, undefined );
+};
+
+/**
+ * Asserts that a select matches the given IDs.
+ * The select is performed with the `#qunit-fixture` context.
+ * @param {String} a - Assertion name
+ * @param {String} b - Sizzle selector
+ * @param {String} c - Array of ids to construct what is expected
+ * @example t("Check for something", "//[a]", ["foo", "bar"]);
+ * @result returns true if "//[a]" return two elements with the IDs 'foo' and 'bar'
+ */
+QUnit.assert.u = function( a, b, c ) {
```

Comment:
With that comment, i have also invited you to this discussion, what do you think?